tests/dummy/app/views/editable-table-cell.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+import TableCell from 'ember-table/views/table-cell';
+
+export default TableCell.extend({
+  className: 'editable-table-cell',
+  templateName: 'editable-table/editable-table-cell',
+  isEditing: false,
+  type: 'text',
+
+  innerTextField: Ember.TextField.extend({
+    typeBinding: 'parentView.type',
+    valueBinding: 'parentView.cellContent',
+    didInsertElement: function() {
+      return this.$().focus();
+    },
+    focusOut: function() {
+      return this.set('parentView.isEditing', false);
+    }
+  }),
+
+  onRowContentDidChange: Ember.observer(function() {
+    return this.set('isEditing', false);
+  }).observes('row.content'),
```

Comment:
This is not the correct syntax for an observer, it mixes the 2 syntaxes ( with and without prototype extension )